GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+# Governance
+
+This document defines the governance process for the GitOps Working Group and community and OpenGitOps project.
+
+## Roles
+
+### All Members
+
+This is an open public working group welcoming anyone who would like to join and help.
+There are no special requirements for membership apart from the responsibilities listed  below.
+Members are not required to have any explicit roles or additional responsibilities, but they may also have formally assigned roles (see below).
+See the Working Group [README](README.md) for information about volunteering for committees and projects.
+Note that voting requires additional conditions (see [Elections](#elections)).
+
+Responsibilities:
+
+- Must follow the [Code of Conduct](CODE_OF_CONDUCT.md)
+- May not create the impression that they have any authority or formal responsibilities within the GitOps WG other than their explicitly assigned roles
+
+### Committees
+
+Working Group [Members](#all-members) may volunteer to join WG Committees and help with specific projects according to their interest and ability.
+Responsibilities vary per committee, but in general please remember everyone is a volunteer.
+Committees follow processes to allow for progress so long as they maintain inclusiveness, transparency, and respect for all involved.
+
+Current WG committees:
+
+- Principles Committee
+  - Responsible for drafting the revised GitOps Principles (see [Timeline](#timeline))
+- Events Committee
+  - Responsible for organizing events such as GitOpsCon, and supporting other events in the wider GitOps community (see [Timeline](#timeline))
```

Comment:
Link to the documents repo, where the principles are now published.

To-do: remove the principles committee in a future update, now that it has folded back into the wider group. My suggestion is to make that change as a separate Git commit, so there's a record.

Remove anchor to the old timeline. GitOps WG timeline moved here (https://github.com/cncf/tag-app-delivery/tree/main/gitops-wg#timeline), but this list on the website is more appropriate to what is described here for the events committee.

```suggestion
  - Responsible for drafting the revised [GitOps Principles](https://github.com/open-gitops/documents)
- Events Committee
  - Responsible for organizing events such as GitOpsCon, and supporting other events in the wider GitOps community (see [OpenGitOps events](https://opengitops.dev/events))
```